### PR TITLE
Mohammad.cpp

### PR DESCRIPTION
**Pull request by haroon73**

### CODE REVIEW

File: Week 1/Weekly Practice Problems (Week 1)/0412: Fizz Buzz/Mohammad.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        }
+        else if(i  % 5 == 0){
+            v.push_back("Buzz");
+
+        }
+        else{
+            stringstream s;
+            s << i;
+            string str = s.str();
+            v.push_back(str);
```
Comment (DominicS99) [inline Week 1/Weekly Practice Problems (Week 1)/0412: Fizz Buzz/Mohammad.cpp]: We can shorten this a bit by using the to_string() function on i
```suggestion
            v.push_back(to_string(i));
```

File: Week 1/Weekly Practice Problems (Week 1)/0412: Fizz Buzz/Mohammad.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class Solution {
+public:
+    vector<string> fizzBuzz(int n) {
+        vector<string> v;
+        for(int i = 1 ; i <= n ; i++){
```
Comment (DominicS99) [inline Week 1/Weekly Practice Problems (Week 1)/0412: Fizz Buzz/Mohammad.cpp]: To do for practice:
Use i for indexing and another variable to tell whether you've reached a clean division of 3 or 5.

**Review by DominicS99 — DISMISSED** [review]
Left some potential changes if you'd like to update your code. I also recommend indenting the code within the for loop, because with larger chunks of code it might be difficult to easily tell where it ends.

File: Week 1/Weekly Practice Problems (Week 1)/0412: Fizz Buzz/Mohammad.cpp [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        }
+        else if(i % 3 == 0 ){
+            v.push_back("Fizz");
+        }
+        else if(i  % 5 == 0){
+            v.push_back("Buzz");
+
+        }
+        else{
+            v.push_back(to_string(i));
```
Comment (haroon73) [inline Week 1/Weekly Practice Problems (Week 1)/0412: Fizz Buzz/Mohammad.cpp]: update this part of the code

**Review by donaldong — APPROVED** [review]
Nice! Thanks

**Pull request merged**